website/docs/docs/develop/getting-started/learning-more/getting-started-dbt-core.md

```diff
@@ -2,16 +2,16 @@
 title: "Getting started with dbt Core"
 id: getting-started-dbt-core
 description: "Connecting your warehouse to dbt Core using the CLI."
-sidebar_label: "Getting started with dbt Core"
+sidebar_label: "Getting started"
```

Comment:
this guide doesn't have an "Overview page" accompaniment and i think it reads all right.